tests/suites/manual/spaces.sh

```diff
@@ -46,7 +46,7 @@ run_spaces_manual_aws() {
 		--filters "Name=name,Values=ubuntu/images/hvm-ssd/ubuntu-${series}-?????-amd64-server-????????" 'Name=state,Values=available' \
 		--query 'reverse(sort_by(Images, &CreationDate))[:1].ImageId' \
 		--output text)
-	if [ -z "${OUT}" ]; then
+	if [[ -z ${OUT} ]]; then
```

Comment:
```suggestion
	if [[ -z "${OUT}" ]]; then
```

Reply:
No, you don't need that because of the `[[ ]]` :)